rs/nns/integration_tests/src/upgrade_canisters_with_golden_nns_state.rs

```diff
@@ -16,7 +16,6 @@ use ic_nns_test_utils::{
     },
 };
 use ic_nns_test_utils_golden_nns_state::new_state_machine_with_golden_nns_state_or_panic;
-use std::str::FromStr;
```

Comment:
Hmm. I am surprised clippy did not block this from being submitted before.

Reply:
I'm surprised as well!